### PR TITLE
fix: messages header

### PR DESCRIPTION
**Pull request by GusBaamonde**
Class name modifications to use the same class name 'BE' brings

### CODE REVIEW

**Comment by cbernat** [comment]
👍

File: assets/templates/messages-header.html [hunk 1]
```diff
@@ -42,9 +42,9 @@
       </ul><!-- nav -->
     </section><!-- section-module -->
     <!-- rojo -->
-    <div class="messages-container error">
+    <div class="messages-container blocker">
       <div class="wrapper">
-        <p class="ng-binding">Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.</p>
```
Comment (cbernat) [inline assets/templates/messages-header.html]: Great!

**Comment by andresmoschini** [comment]
👍 Good work!

Please test this code with @cbernat before merging and closing the issue.

**Comment by cbernat** [comment]
@andresmoschini tested locally and works fine. Here is the proof:
image

**Pull request merged**